### ISSUE TITLE
exec_module() missing 1 required positional argument: 'module'

### ISSUE THREAD

**Issue by radarhere** [pypa/setuptools#2246]
Hi. Over at https://github.com/python-pillow/Pillow/issues/4769, we've started receiving an error in Python 3.9 -

```
TypeError: exec_module() missing 1 required positional argument: 'module'
```

I find that our code passes with setuptools 47.3.1, but fails with 47.3.2. Investigating, I find that https://github.com/pypa/setuptools/pull/2071/files adds `ExtensionFileLoader(__name__,__file__).exec_module()` - calling `exec_module` without any arguments.

Looking at https://docs.python.org/3/library/importlib.html#importlib.machinery.ExtensionFileLoader, I would conclude that yes, `exec_module` does require an argument.

I'm hoping that seems like a suitably obvious bug in setuptools. Let me know if it's not.

**Comment by jaraco** [comment]
@alexhenrie Can you investigate and devise a fix?

**Comment by alexhenrie** [comment]
Shoot, you have my apologies. I have opened pull request #2249 which I believe will get everything working again.

**Comment by danielgordon10** [comment]
Can you please push a patch to PYPI? Many projects that don't pin the version are starting to get the bug.

**Comment by jaraco** [comment]
The fix was released in 49.1.3 and 49.2.0, both in PyPI since Jul 12.

<img width="815" alt="image" src="https://user-images.githubusercontent.com/308610/88444615-fb93d680-cdeb-11ea-98ef-9912e31f71d0.png">

Are projects not pinning Setuptools getting those slighly older than latest versions?

**Comment by danielgordon10** [comment]
https://github.com/numpy/numpy/pull/16870/files#diff-522adf759addbd3b193c74ca85243f7d For example NumPY doesn't trust 49.2.0
